SU-HSDO/suhumsci
# feat(STN-337):  tradl theme work

## Summary
More work to get tradl theme to have colorful logic, HTML, and CSS as base styles wherever possible so everything works out of the box.

TODO
- [x] some regression testing with colorful theme
- [x] compiled css

Note: heros are not working yet. it feelsl like a complex one since they vary a lot theme to theme.

## Steps to Test
1. In humsci_basic, `npm run test` passes.
2. Check the site. Should look very similar to Sand Colorful theme pairing, except for headings, and some link and button styles. Everything should work out of the box.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
